test/e2e_node/dra_test.go

```diff
@@ -881,8 +881,7 @@ func createTestObjects(ctx context.Context, clientSet kubernetes.Interface, node
 			RestartPolicy: v1.RestartPolicyNever,
 		},
 	}
-	switch podName {
-	case "drasleeppod":
+	if podName == "drasleeppod" {
```

Comment:
No strong opinion about this one.

Reply:
The same thing. I wouldn't touch it if CI is green.